src/tests/offers_api.test.ts

```diff
@@ -319,4 +320,165 @@ describe('Offers API', () => {
       expect(res.errors?.[0].message).toEqual('Not permitted to update offer')
     })
   })
+
+  describe('listOffers', () => {
+    let otherOfferInShipment: Offer,
+      otherShipment: Shipment,
+      otherShipmentOffer: Offer
+
+    const LIST_OFFERS_QUERY = gql`
+      query($shipmentId: Int!) {
+        listOffers(shipmentId: $shipmentId) {
+          id
+          status
+          shipmentId
+          sendingGroupId
+          photoUris
+          contact {
+            name
+            email
+            signal
+          }
+        }
+      }
+    `
+
+    beforeEach(async () => {
+      otherShipment = await createShipment({
+        labelYear: 2021,
+        labelMonth: 1,
+        shippingRoute: ShippingRoute.Uk,
+        sendingHubId: captainsGroup.id,
+        receivingHubId: captainsGroup.id,
```

Comment:
Question: would this be possible in the "real world"? Having the same sending and receiving hub? I wonder if we should validate that on the backend. On the frontend, we split shipping groups by `groupType` so that this can never happen.

Reply:
There's nothing preventing it from happening in the real world if an admin decides to do it. Since only admins can mutate shipments though I'm less worried about locking this down with rigorous consistency checks.